includes/admin/meta-boxes/class-wc-meta-box-order-actions.php

```diff
@@ -33,7 +33,7 @@ public static function output( $post ) {
 		}
 
 		$order_actions = apply_filters( 'woocommerce_order_actions', array(
-			'send_order_details'              => __( 'Email order details to customer', 'woocommerce' ),
+			'send_order_details'              => __( 'Email customer invoice / order details to customer', 'woocommerce' ),
```

Comment:
I guess we could simplify to:

> Email invoice / order details to customer

Reply:
We could, I just thought it makes sense to keep it consistent cause it was originally called `Customer invoice` in the email class title. I did not want to change it completely so that it is unrecognizable to customers using the functionality already.

I am happy to change it to `Invoice / Order details` if everyone else is in agreement?

Reply:
Either would work for me, I have a slight preference for `customer invoice / order details` to keep consisency, even when that's a longer string. May also avoid translation issues (easier to make it match).